unit_tests/speech/_fixtures.py

```diff
@@ -0,0 +1,12 @@
+SYNC_RECOGNIZE_RESPONSE = {
```

Comment:
1. Done
2. I also put it in there, because the number of variables will include if we add more functionalities (e.g. for StreamingRecognition)